ui/components/app/import-token/token-search/token-search.component.js

```diff
@@ -12,12 +12,14 @@ export default class TokenSearch extends Component {
 
   static defaultProps = {
     error: null,
+    searchClassName: undefined,
   };
 
   static propTypes = {
     onSearch: PropTypes.func,
     error: PropTypes.string,
     tokenList: PropTypes.object,
+    searchClassName: PropTypes.string,
```

Comment:
I don't see where this `searchClassName` prop is being used.

Reply:
It is used in `import-tokens-modal.js` file ; `searchClassName="import-tokens-modal__button-search"` 
to make the dix outline blue color